cmd/istio-cni/main.go

```diff
@@ -227,6 +227,11 @@ func cmdAdd(args *skel.CmdArgs) error {
 					if redirect, redirErr := NewRedirect(ports, annotations); redirErr != nil {
 						log.Errorf("Pod redirect failed due to bad params: %v", redirErr)
 					} else {
+						if redirect.includePorts != "*" {
+							log.Errorf("lambdai: should always redirect to local port but see %v", redirect.includePorts)
```

Comment:
Remove `lambdai` here.

Reply:
Aha, this should be entirely removed. It is possible to allow redirect individual port.

Reply:
Print redirected port at info level.
Thank you!